plugins/rikki/heroeslounge/components/ScheduleMatch.php

```diff
@@ -69,21 +64,20 @@ public function onSaveDate()
         $match = Match::find(post('match_id'));
         if ($match) {
             $date = post('date');
-            $timezone = post('timezoneName');
+            $timezone = TimezoneHelper::getTimezone();
             if ($date != null) {
                 try {
                     $x = new DateTime($date, new DateTimeZone($timezone));
-                    $x->setTimezone(new DateTimeZone('Europe/Berlin'));
+                    $x->setTimezone(new DateTimeZone(TimezoneHelper::defaultTimezone()));
```

Comment:
This needs to be the server timezone (UTC)

Reply:
Yeah that's why the default timezone pulls from config/app